src/components/Tabs/Tabs.less

```diff
@@ -31,30 +31,74 @@
 		}
 
 		&:hover {
-			background-color: @color-gray-5;
+			background-color: @color-lightGray;
+			.Tabs-mixin-tab-arrow-background(@color-lightGray);
 		}
 
 		&:active {
-			box-shadow: @shadow-inset;
+			background-image: linear-gradient(#dbdbdb 0, @color-lightGray @Tabs-spacer-height);
 		}
 	}
-	&-Tab-is-active {
-		border-top: @Tabs-spacer-height solid @color-primary;
-		padding-top: 0;
-		color: #000;
-		cursor: initial;
 
+	&-Tab-is-disabled {
+		color: fade(#333, 50%);
```

Comment:
good point.  will change when I clean up some of the other variables issues that we have